src/radio-group.test.interactions.ts

```diff
@@ -238,7 +238,6 @@ it('updates `value` when the `value` of a checked radio is set programmatically'
 
   assert(radio);
   radio.value = 'three';
-  await host.updateComplete;
```

Comment:
No longer needed [not](https://github.com/CrowdStrike/glide-core/pull/764/files#diff-1bcd1e1e7387a7e90565943f734129e101c4a9ae323d3ef939d187ebc80be046L155) that we're not waiting for `updateComplete` in the `value` setter.